composer.json

```diff
@@ -37,17 +37,17 @@
     ],
 
     "require": {
-        "php": ">=5.5",
-        "league/climate": "^2.0|^3.0",
+        "php": "^5.6 || ^7.0",
+        "league/climate": "^2.0 || ^3.0",
         "symfony/console": "^2.0",
         "symfony/process": "^2.1"
     },
 
     "require-dev": {
         "mockery/mockery": "^0.9",
-        "phpunit/phpunit": "^4.0",
+        "phpunit/phpunit": "^5.1",
         "sensiolabs/security-checker": "^3.0",
-        "squizlabs/php_codesniffer": "^2.0"
+        "squizlabs/php_codesniffer": "^2.5"
```

Comment:
What about keeping lowest possible version in `composer.json` for using project as dep, and then keep highest possible version in `composer.lock` for using project as standalone tool?

And even then - I would see no great gain of having lock file in repo